### ISSUE TITLE
Use from python

### ISSUE THREAD

**Issue by jschrewe** [aaronsw/html2text#20]
I'm not sure if this is really an issue but if I try to use html2text from a python program it fails with `Caught NameError while rendering: global name 'options' is not defined`.

What I'm trying to do is:

```
import html2text

def html2markdown(html):
    return html2text.html2text(html)
```

Is this even possible?

**Comment by aaronsw** [comment]
Sorry about that -- looks like some changes I pulled yesterday broke
imports. It should be fixed in the latest commit.

**Comment by aaronsw** [comment]
Fixed in 76a353fcd0.
